From: Jisheng Zhang <jszhang@marvell.com>
Date: Wed, 16 Mar 2016 17:59:40 +0800
Subject: PCI: mvebu: Constify mvebu_pcie_pm_ops structure
Git-commit: 6e9a426d7964703098bde5752ae5d062a596ca99
Patch-mainline: v4.7-rc1
References: fate#319899 fate#320700

The mvebu_pcie_pm_ops structure is never modified, so declare it as const.

Signed-off-by: Jisheng Zhang <jszhang@marvell.com>
Signed-off-by: Bjorn Helgaas <bhelgaas@google.com>
Reviewed-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Signed-off-by: Matthias Brugger <mbrugger@suse.com>
---
 drivers/pci/host/pci-mvebu.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/pci/host/pci-mvebu.c b/drivers/pci/host/pci-mvebu.c
index 53b79c5..6c43cb8 100644
--- a/drivers/pci/host/pci-mvebu.c
+++ b/drivers/pci/host/pci-mvebu.c
@@ -1298,7 +1298,7 @@ static const struct of_device_id mvebu_pcie_of_match_table[] = {
 };
 MODULE_DEVICE_TABLE(of, mvebu_pcie_of_match_table);
 
-static struct dev_pm_ops mvebu_pcie_pm_ops = {
+static const struct dev_pm_ops mvebu_pcie_pm_ops = {
 	.suspend_noirq = mvebu_pcie_suspend,
 	.resume_noirq = mvebu_pcie_resume,
 };
-- 
2.10.0

